src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@amarlearning Just tried it. It doesn't work. Keep in mind that the icon is not actually just another letter, its in a CSS pseudo-element.